variables.tf

```diff
@@ -28,13 +28,13 @@ variable "system_user" {
 
 variable "use_ssh_agent" {
   type        = bool
-  default     = "true"
+  default     = "false"
   description = "Whether to use ssh agent"
 }
 
 variable "write_kubeconfig" {
   type        = bool
-  default     = "false"
+  default     = "true"
   description = "Write kubeconfig file to disk"
```

Comment:
Let's keep original values. I prefer not rely on ssh/scp by default as it's an ugly hack.

Reply:
done